WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -208,6 +208,11 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 
 			if ( T_OPEN_PARENTHESIS === $this->tokens[ $i ]['code'] ) {
 
+				if ( ! isset( $this->tokens[ $i ]['parenthesis_closer'] ) ) {
+					// Live coding or parse error.
+					break;
```

Comment:
Usually when PHPCS's core sniffs detect a parse error, they don't just skip out, they add a warning. At least they do in some cases. Should we consider doing the same here?